org.cispa.Scyther.json

```
@@ -0,0 +1,139 @@
+{
+   "app-id":"org.cispa.Scyther",
+   "runtime":"org.freedesktop.Platform",
+   "runtime-version":"19.08",
```

Update runtime to 20.08
```suggestion
   "runtime-version":"20.08",
```